<launch>
    <incldue file="$(find socketcan_bridge)/launch/socketcan_bridge.launch"/>
    <incldue file="$(find pix_driver)/launch/pix_driver_remote.launch"/>
    <node pkg="remote_control" name="remote_server" type="server_test_1.py" output="screen"/>
    <node pkg="remote_control" name="remote_control" type="control_converter_speed.py" /> 
    <!-- <node pkg="remote_control" name="acu_sweepctrlcmd" type="acu_sweepctrlcmd.py" />  -->
</launch>
